### PR TITLE
Django110

### PR DESCRIPTION
**Pull request by brad**
@orcasgit/orcas-developers Please review these updates to make fernet fields work in Django 1.10. The only change needed was to avoid using the model method `get_prep_lookup` which has [been removed](https://docs.djangoproject.com/en/1.10/releases/1.10/#field-get-prep-lookup-and-field-get-db-prep-lookup-methods-are-removed)

Following [the docs](https://docs.djangoproject.com/en/1.10/releases/1.10/#field-get-prep-lookup-and-field-get-db-prep-lookup-methods-are-removed), I reused our existing code to register lookups for all the standard field lookups (`__exact`, `__gte`, `__isnull`, etc...)

I also reconfigured `tox.ini` to be more current and support a simplified `.travis.yml`

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,8 +1,11 @@
 [tox]
 envlist =
+    py36-{docs,flake8},
     py{27,33,34,py,py3}-django18-{pg,sqlite},
-    py{27,34,35,py}-djangotrunk-{pg,sqlite},
-    py{27,py}-django{18,trunk}-mysql,
+    py{27,34,35,py}-django{19,110}-{pg,sqlite},
+    py{27,34,35,36,py}-django111-{pg,sqlite},
+    py{35,36}-djangotrunk-{pg,sqlite},
+    py{27,py}-django{18,19,110,111}-mysql,
```
Comment (grokcode) [inline tox.ini]: @brad 👍  I'm glad we can continue supporting 1.8 and 1.9 without any special cases.

File: fernet_fields/fields.py [hunk 2]
```diff
@@ -100,6 +90,22 @@ def validators(self):
...
+def get_prep_lookup(self):
+    """Handles lookups for EncryptedField fields"""
+    if self.lookup_name == 'isnull':
+        return super(self.__class__, self).get_prep_lookup()
+    else:
+        raise FieldError("Encrypted fields do not support lookups.")
+
+
+# Register all field lookups to our handler
+for name, lookup in models.Field.class_lookups.items():
```
Comment (grokcode) [inline fernet_fields/fields.py]: @brad I think we can simplify things slightly by not registering a new lookup for `isnull`, then we can remove the `if`/`else` in `get_prep_lookup` and just raise the `FieldError` in all cases.

File: fernet_fields/fields.py [hunk 3]
```diff
@@ -100,6 +90,22 @@ def validators(self):
             del self.__dict__['_internal_type']
 
 
+def get_prep_lookup(self):
+    """Handles lookups for EncryptedField fields"""
+    if self.lookup_name == 'isnull':
+        return super(self.__class__, self).get_prep_lookup()
+    else:
+        raise FieldError("Encrypted fields do not support lookups.")
```
Comment (grokcode) [inline fernet_fields/fields.py]: @brad Shoot, it is helpful to have the class name and field name in the error message. I'm wondering if there is a way we can keep that?

File: fernet_fields/fields.py (hunk 2 above)
Comment (brad) [inline fernet_fields/fields.py]: @grokcode Good call! I will try that

**Comment by grokcode** [comment]
LGTM! :shipit:

**Pull request merged**